decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -100,6 +100,8 @@ def install_decidim_webpacker
 
         # Run Decidim custom procfile installation
         rails "decidim:procfile:install"
+        # replace robots.txt
+        rails "decidim:robots:replace"
```

Comment:
Empty line and capitalize to make it consistent with the rest of the method: 

```suggestion
        rails "decidim:procfile:install"

        # Replace robots.txt
        rails "decidim:robots:replace"
```